src/main/java/frc/lib5k/components/sensors/ColorSensor.java

```diff
@@ -0,0 +1,427 @@
+/*
```

Comment:
Please don't just copy-paste other libraries into ours. This is why @override exists.